Makefile

```diff
@@ -10,46 +24,79 @@ DISTDIR := $(shell mktemp -d --tmpdir=$(BUILDDIR) dist-XXXXXXXX)
 endif
 endif
 
+$(shell $(call reinit_repo,.config/python-style,$(PYTHON_STYLE_HEAD)))
+$(shell $(call reinit_repo,contrib/gnatcoll-bindings,$(GNATCOLL_HEAD)))
+$(shell $(call reinit_repo,contrib/langkit,$(langkit_HEAD)))
+
 export MYPYPATH = $(PWD)/stubs
 
-python-packages := language tests disttools/setup.py disttools/gprgen.py
-
-.PHONY: all check check_black check_isort check_flake8 check_pylint check_mypy check_pydocstyle format \
-	test test_python test_python_coverage install install_devel install_devel_edge clean
+.PHONY: all
 
 all: check test
 
+.PHONY: init deinit
+
+init: .config/python-style contrib/gnatcoll-bindings contrib/langkit
+	$(VERBOSE)$(call checkout_repo,.config/python-style,$(PYTHON_STYLE_HEAD))
+	$(VERBOSE)$(call checkout_repo,contrib/gnatcoll-bindings,$(GNATCOLL_HEAD))
+	$(VERBOSE)$(call checkout_repo,contrib/langkit,$(LANGKIT_HEAD))
+	$(VERBOSE)rm pyproject.toml
+	$(VERBOSE)ln -s .config/python-style/pyproject.toml
```

Comment:
Removing and linking could be combined into `ln -sf .config/python-style/pyproject.toml pyproject.toml`

Reply:
Changed.